pkg/dom0-ztools/rootfs/bin/eve

```diff
@@ -118,6 +119,9 @@ __EOT__
                  ;;
           esac
           ;;
+ version)
+          echo $(cat /run/eve-release)
```

Comment:
echo $() was used to ensure line break (/run/eve-release doesn't have line break).

Reply:
Either add the shellcheck disable comment, or do e.g.,
 v=$(cat /run/eve-release)
echo "$v"